### ISSUE TITLE
How to change matching brace color in Geany v2.0 ?

### ISSUE THREAD

**Issue by that-ben** [geany/geany#3680]
Might be related to or even a duplicate of #885 but I could not figure it out.  I'm new to Geany.

<img width="280" alt="Screenshot 2023-11-10 at 1 08 55 PM" src="https://github.com/geany/geany/assets/13210399/282df898-199e-4826-985b-dee5058c3167">

If it means anything, I'm currently using the "Kary Pro Colors - Bare Dark" scheme.

**Comment by elextr** [comment]
Same as #885, to change a colour scheme you have to edit the colour scheme file you are using and change the setting #885 mentioned.

**Comment by that-ben** [comment]
Alright, so given that I'm using the "Kary Pro Colors - Bare Dark" scheme, I can still launch this to access the file you're talking about, correct?
 
<img width="1363" alt="Screenshot 2023-11-10 at 5 32 16 PM" src="https://github.com/geany/geany/assets/13210399/4a4bef79-1872-4437-887e-8749c3ab9e0b">

**Comment by elextr** [comment]
No, `filetypes.common` is the default colour scheme, you need whatever file you installed for the one you are using.

**Comment by that-ben** [comment]
I did not install any, I selected "Kary Pro Colors - Bare Dark" from list of bundled schemes like this:

<img width="1153" alt="Screenshot 2023-11-10 at 5 41 11 PM" src="https://github.com/geany/geany/assets/13210399/6f08511a-1fa9-43d3-8d37-21da34a408b8">

I'm sorry if I'm infuriating long time users with this question right here, but I'm a complete Geany n00b so I have no clue what I'm doing! 🤣 Where should I look for that file exactly?

**Comment by elextr** [comment]
What OS are you using?

**Comment by that-ben** [comment]
macOS Sonoma on a brand new iMac M3 that I received yesterday.

**Comment by elextr** [comment]
Oh, apples are things I eat, not compute so I can't help you.  But there is a mac specific issues for Geany so I'll transfer this issue to there.
